DFC.ServiceTaxonomy.VersionComparison/Controllers/AdminController.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DFC.ServiceTaxonomy.VersionComparison.Models;
+using DFC.ServiceTaxonomy.VersionComparison.Services;
+using DFC.ServiceTaxonomy.VersionComparison.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Localization;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using OrchardCore.ContentManagement;
+using OrchardCore.DisplayManagement;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Notify;
+
+namespace DFC.ServiceTaxonomy.VersionComparison.Controllers
+{
+    public class AdminController : Controller
+    {
+        private readonly IAuditTrailQueryService _auditTrailQueryService;
+        private readonly IContentManager _contentManager;
+        private readonly IShapeFactory _shapeFactory;
+        private readonly IDisplayManager<VersionComparisonOptions> _versionComparisonOptionsDisplayManager;
+        private readonly IUpdateModelAccessor _updateModelAccessor;
+        private readonly INotifier _notifier;
+        private readonly IHtmlLocalizer<AdminController> _h;
+
+        public AdminController(
+            IAuditTrailQueryService auditTrailQueryService,
+            IContentManager contentManager,
+            IDisplayManager<VersionComparisonOptions> versionComparisonOptionsDisplayManager,
+            IHtmlLocalizer<AdminController> htmlLocalizer,
+            INotifier notifier,
+            IShapeFactory shapeFactory,
+            IUpdateModelAccessor updateModelAccessor
+        )
+        {
+            _auditTrailQueryService = auditTrailQueryService;
+            _contentManager = contentManager;
+            _h = htmlLocalizer;
+            _notifier = notifier;
+            _shapeFactory = shapeFactory;
+            _updateModelAccessor = updateModelAccessor;
+            _versionComparisonOptionsDisplayManager = versionComparisonOptionsDisplayManager;
+        }
+
+        public async Task<ActionResult> Index(string contentItemId)
```

Comment:
Should stick a [HttpGet] here?

Reply:
Not usually done in OC core code as is implied as a GET by default but have added in our module so is explicit.